web/src/Routes.tsx

```diff
@@ -35,11 +35,13 @@ const Routes = () => {
           <Route path="/deactivated" page={DeactivatedPage} name="deactivated" />
         </Private>
         <Private unauthenticated="home">
-          <Private unauthenticated="deactivated" roles={['user', 'admin']}>
+          <Private unauthenticated="deactivated" roles={['user', 'admin', 'pioneer']}>
             <Route path="/territory/{id}" page={TerritoryPage} name="territory" />
             <Route path="/my-territories" page={MyTerritoriesPage} name="myTerritories" />
             <Route path="/user-account" page={UserAccountPage} name="userAccount" />
-            <Route path="/self-checkout" page={SelfCheckoutPage} name="selfCheckout" />
+            <Private unauthenticated='deactivated' roles={['user', 'admin']}>
```

Comment:
Ah wait when this is not authenticated, it should go to my territories?

Reply:
I could make it go there, however, I was thinking about making a "You don't have access to this" page later. I didn't see it as a problem right now because I think the only way for a pioneer to get to the self-checkout is by manually typing it in the URL because the options in the UI are gone for pioneers.